### ISSUE TITLE
Make clear what versions of Coq are supported

### ISSUE THREAD

**Issue by robbertkrebbers** [xavierleroy/canonical-binary-tries#2]
This information is not in the README. I think it would be good to include it there.

**Comment by xavierleroy** [comment]
Fair point.  I admit I have no idea :-)   Probably we should copy&paste the CI from other Coq libraries, that's a good way to test multiple Coq versions.

**Comment by robbertkrebbers** [comment]
> Probably we should copy&paste the CI from other Coq libraries, that's a good way to test multiple Coq versions.

That would be great, especially if people want to rerun your benchmarks to compare with their own map implementation it's useful if are guaranteed to work with a recent Coq.
